libnavigation-core/src/test/java/com/mapbox/navigation/core/trip/session/MapboxTripSessionTest.kt

```diff
@@ -173,6 +175,8 @@ class MapboxTripSessionTest {
         mockkObject(ThreadController)
         every { ThreadController.getIOScopeAndRootJob() } returns JobControl(parentJob, testScope)
         every { ThreadController.getMainScopeAndRootJob() } returns JobControl(parentJob, testScope)
+        every { routeProgress.bannerInstructions() } returns null
```

Comment:
Without adding the additional "every" call(s) I get stack traces when running the tests like: "Exception in thread "main @coroutine#7" io.mockk.MockKException: no answer found for: RouteProgress(#55).bannerInstructions()..."

While the tests still pass I thought it better to remove the stack traces since they can imply an error occurred when it didn't.  In the past versions of mockk the test would fail. My thinking at the time was that it cleans up the testing output while not negatively impacting the tests. However if you feel strongly that the calls should be removed I can remove them. Please let me know.